vmdb/spec/tools/environment_builders/openstack/network_methods.rb

```diff
@@ -101,32 +99,34 @@ def create_router_neutron(data)
 
     # TODO: enhance Fog Neutron router support
     puts "Routers must be created manually.  Please run:"
-    puts "  neutron router-create #{data[:name]}"
+    puts "  neutron router-create --tenant-id #{@project.id} #{data[:name]}"
     puts "  neutron router-gateway-set #{data[:name]} #{@network_public.name}"
     puts "  neutron router-interface-add #{data[:name]} #{@subnet_private.name}"
     exit 1
   end
 
-  def find_floating_ip_neutron(data)
+  def find_floating_ip_neutron
     collection = fog_network.floating_ips
     puts "Finding address in #{collection.class.name}"
-    collection.detect { |i| i.floating_ip_address == data[:floating_ip_address] }.try(:floating_ip_address)
+    collection.detect { |i| i.tenant_id == @project.id }.try(:floating_ip_address)
   end
 
-  def create_floating_ip_neutron(data)
+  def create_floating_ip_neutron
     collection = fog_network.floating_ips
     puts "Creating address against #{collection.class.name}"
-    collection.create(data.merge(:floating_network_id => @network_public.id)).floating_ip_address
+    collection.create(:floating_network_id => @network_public.id, :tenant_id => @project.id).floating_ip_address
   end
 
   def find_floating_ip_nova(data)
     collection = fog_network.addresses
+    data       = settings[:network][:floating_ip]
     puts "Finding address in #{collection.class.name}"
     collection.detect { |i| i.ip == data[:ip] }.try(&:ip)
   end
 
   def create_floating_ip_nova(data)
     collection = fog_network.addresses
+    data       = settings[:network][:floating_ip]
```

Comment:
Same question regarding is the data argument needed or handle correctly when/if provided?